Attribution.txt

```diff
@@ -202,3 +202,6 @@ https://github.com/hashicorp/errwrap/blob/master/LICENSE
 
 hashicorp/go-multierror (Mozilla Public License 2.0) https://github.com/hashicorp/go-multierror
 https://github.com/hashicorp/go-multierror/blob/master/LICENSE
+
+github.com/go-kit/log (MIT) https://github.com/go-kit/log
+https://github.com/go-kit/log/blob/master/LICENSE
```

Comment:
The go-kit section can also be removed

Reply:
Looks to have come from this change in contracts: https://github.com/edgexfoundry/go-mod-core-contracts/commit/6ac863c88929d69ae2a3023fe4df8833f6f775c5